runtime/lua/provider/node/health.lua

```diff
@@ -0,0 +1,106 @@
+local health = vim.health
+
+local iswin = vim.loop.os_uname().sysname == 'Windows_NT'
+
+local M = {}
+
+function M.check()
+  health.start('Node.js provider (optional)')
+
+  if health.provider_disabled('node') then
+    return
+  end
+
+  if
+      not vim.fn.executable('node') == 1
+      or (vim.fn.executable('npm') == 0 and vim.fn.executable('yarn') == 0 and vim.fn.executable('pnpm') == 0)
+  then
+    health.warn(
+      '`node` and `npm` (or `yarn`, `pnpm`) must be in $PATH.',
+      'Install Node.js and verify that `node` and `npm` (or `yarn`, `pnpm`) commands work.'
+    )
+    return
+  end
+
+  -- local node_v = vim.fn.split(system({'node', '-v'}), "\n")[1] or ''
+  local ok, node_v = health.cmd_ok({ 'node', '-v' })
```

Comment:
```suggestion
  -- local node_v = vim.fn.split(system({'node', '-v'}), "\n")[1] or ''
  local ok, node_v = health.cmd_ok({ 'node', '-v' })
```

As it's commented out, I guess not splitting the lines was deliberate? The original code splits and just uses the 0th element as `node_v` if it exists, else "".